app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to @user
+    else
```

Comment:
- [ ] Kindly as per the requirements, when a user gets saved, it should redirect to new_user_path ⬇️ 
![user_create](https://user-images.githubusercontent.com/32361963/113701797-4a1aa400-96e1-11eb-8d20-f7b7fc66538c.png)

Reply:
Add change like requested